backend/tournesol/serializers/poll.py

```diff
@@ -103,6 +103,29 @@ class Meta:
         read_only_fields = fields
 
 
+class RecommendationRandomSerializer(serializers.Serializer):
```

Comment:
I'm slightly worried about the code duplication. This Serializer is nearly exactly the same thing as RecommendationSerializer defined above.

Also the name is not very well chosen. "Random" relates to how the entities are selected not to what data we want to include in the response.

Reply:
That's right. I first thought about adding specific fields in this serializer, but I finally decided to keep it simple.

I can make a "base" serializer out of it, and `RecommendationSerializer` could simply extend it with its own metadata.

Reply:
Addressed by https://github.com/tournesol-app/tournesol/pull/1876/commits/cd27180532c252b68ffd80b3757691172a4ab68a

`RecommendationBaseSerializer` is a base class containing all fields a recommendation serializer should always provide: the data related to the serialized entity, its context, and the collective ratings.

`RecommendationSerializer` inherits from `RecommendationBaseSerializer` and adds ts own metadata.

The inner class `Meta` of `RecommendationSerializer` is explicitly defined as recommended by the DRF documentation, but we could make it inherit from `RecommendationBaseSerializer` if we prefer.

`RecommendationSerializer` is not a model serializer anymore, it's now a regular serializer. This serializer is a representation built from several different models, and is not designed to be able to create or update a specific Django model.